confdb/aclocal_cc.m4

```diff
@@ -560,56 +562,48 @@ if test "$enable_strict_done" != "yes" ; then
         -Wstack-usage=262144
     "
 
-    enable_c89=no
-    enable_c99=yes
-    enable_posix=2001
+    add_cflags=yes
+    c_std=c99
+    posix_std=2001
     enable_opt=yes
     flags="`echo $1 | sed -e 's/:/ /g' -e 's/,/ /g'`"
     for flag in ${flags}; do
         case "$flag" in
-	     c89)
-		enable_strict_done="yes"
-		enable_c89=yes
-                enable_c99=no
+	     stdc89)
+	        c_std=c89
 		;;
-	     c99)
-		enable_strict_done="yes"
-                enable_c89=no
-		enable_c99=yes
+	     stdc99)
+	        c_std=c99
+		;;
+	     stdgnu99)
+	        c_std=gnu99
+		;;
+	     !stdc)
+		c_std=none
```

Comment:
Having consistent indentation here would make it easier to read.

Reply:
Is something inconsistent?  I can't seem to find anything wrong.

Reply:
Hmm. When I look on the web here, the indentation is off. When it shows up in my email, it looks fine. Whatever...don't worry about it.